MainModule/Server/Core/Logs.lua

```diff
@@ -39,21 +38,27 @@ return function(Vargs)
 		Chats = {};
 		Joins = {};
 		Script = {};
+		Replications = {};
+		NetworkOwners = {};
 		RemoteFires = {};
 		Commands = {};
 		Exploit = {};
 		Errors = {};
+		ServerDetails = {};
 		TempUpdaters = {};
-		
 		TabToType = function(tab)
 			local indToName = {
 				Chats = "Chat";
 				Joins = "Join";
 				Script = "Script";
+				Replications = "Replication";
+				PrivateMessages = "PrivateMessages";
+				NetworkOwners = "NetworkOwner";
 				RemoteFires = "RemoteFire";
 				Commands = "Command";
 				Exploit = "Exploit";
 				Errors = "Error";
+				ServerDetails = "ServerDetails"
```

Comment:
Server details in the logs?

Reply:
I think you are sending too many comments about the same thing? Server Details are now as a log to better access the table Logs.ServerDetails

Reply:
Yeah I think I have.